lib/ingram_micro/elements/message_header_no_pw.rb

```diff
@@ -16,8 +16,12 @@ def defaults
   end
 
   def valid?
-    raise IngramMicro::InvalidType.new('message_id must be a number') unless integer?(@element[:message_id])
-    raise IngramMicro::MissingField.new('partner_name must be present') unless @element[:partner_name]
+    unless integer?(@element[:message_id])
```

Comment:
I don't mind it when there is only one condition, but I understand hating in all scenarios as well